package.json

```diff
@@ -390,7 +390,6 @@
     "@storybook/addon-essentials": "^6.0.26",
     "@storybook/addon-knobs": "^6.0.26",
     "@storybook/addon-storyshots": "^6.0.26",
-    "@storybook/addons": "^6.0.16",
```

Comment:
Was this removed on purpose?

Reply:
Sorry I meant to leave a comment. This is a duplicate entry of https://github.com/elastic/kibana/pull/92666/files/750f561bf12c22f12b3864717900d42842641095#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R386

Reply:
Oh! I should have scrolled up 😄